x-pack/plugins/security_solution/cypress/tasks/common.ts

```diff
@@ -60,3 +62,8 @@ export const reload = (afterReload: () => void) => {
   cy.contains('a', 'Security');
   afterReload();
 };
+
+export const cleanKibana = () => {
+  cy.exec(`curl -XDELETE ${Cypress.env('ELASTICSEARCH_URL')}/.kibana_2 -k`);
```

Comment:
Use of `.kibana_2` here seems brittle, especially given that the `empty_kibana` archive itself loads `.kibana`. I believe that means that the original `.kibana` index still remains after this?

I just noticed that the esArchiver service has a `emptyKibanaIndex` method; I know we can't use that within cypress, but it does reinforce the idea of deleting against a pattern.

 Perhaps we should generalize this to `.kibana*` for now?

Reply:
I tried to generalize it but was not working for me, so I used the fastest solution, any idea about how to fix it, is welcomed :)

Reply:
Solved with @rylnd :D